storage/remote/wal_watcher_test.go

```diff
@@ -171,7 +171,7 @@ func TestReadToEndNoCheckpoint(t *testing.T) {
 	err = os.Mkdir(wdir, 0777)
 	testutil.Ok(t, err)
 
-	w, err := wal.NewSize(nil, nil, wdir, 128*pageSize)
+	w, err := wal.NewSize(nil, nil, wdir, 128*pageSize, true)
```

Comment:
is this intentional for making the wal compressed?

Reply:
I wanted the tests to run using the compressed WAL since that is the future. Uncompressed is the default right now though so I could change them to false for now?

Reply:
I would probably make sense to test both cases here. I do agree compressed should some day become the default, but for now uncompressed is the default and the default should be tested.

Reply:
I don't mind either way.

Reply:
(or for this PR I'd be ok with just testing the default and adding the compressed case as a follow up)

Reply:
Adding the compressed case is pretty easy will be done in a couple minutes. If it looks too complicated I can quickly revert it.

Reply:
Added t.Run statements for both compressed and uncompressed.